### PR TITLE
[TS] LPS-191497 Updating spring-boot-starter-oauth2-resource-server to 2.7.11

### PR DESCRIPTION
**Pull request by IstvanD**
Updating **spring-boot-starter-oauth2-resource-server** from **2.7.9** to **2.7.11** in order to fix vulnerability CVE-2023-20862.

Please let me know if the Frontend Infrastructure Team is not the correct team to send pull requests concerning Client Extension.

Thank you and best regards,
István

### CODE REVIEW

**Comment by mateusralv** [comment]
Hi @IstvanD The failures are not related! 

[Testray Comparison](https://testray.liferay.com/home/-/testray/runs/compare?formDate=1692729458934&testrayRunIdA=2408641802&testrayRunIdB=2408671043)

File: modules/util/client-extension-util-spring-boot/build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 dependencies {
 	api group: "org.springframework.boot", name: "spring-boot-starter-oauth2-client", version: "2.7.9"
-	api group: "org.springframework.boot", name: "spring-boot-starter-oauth2-resource-server", version: "2.7.9"
+	api group: "org.springframework.boot", name: "spring-boot-starter-oauth2-resource-server", version: "2.7.11"
```
Comment (gamerson) [inline modules/util/client-extension-util-spring-boot/build.gradle:3]: We should bump the rest of the spring boot references to 2.7.11 as well to make sure everything is still compatible.

Comment (IstvanD) [reply modules/util/client-extension-util-spring-boot/build.gradle:3]: @gamerson I have sent a new pull that updates all spring-boot's libraries from 2.7.9 to 2.7.11
https://github.com/liferay-frontend/liferay-portal/pull/3616

Comment (bryceosterhaus) [reply modules/util/client-extension-util-spring-boot/build.gradle:3]: I will leave https://github.com/liferay-frontend/liferay-portal/pull/3616 open for now, but for future PRs like this feel free to send to @gamerson directly as his team is the maintainers of these portion of client extensions.

Comment (IstvanD) [reply modules/util/client-extension-util-spring-boot/build.gradle:3]: Hi @bryceosterhaus I have sent the pull request to @gamerson 
https://github.com/gamerson/liferay-portal/pull/622

**Pull request closed without merging**